From 926136c92b44bb1b11f9355b7b9df59b3de28aa6 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Sun, 17 Jul 2016 18:14:38 +0300
Subject: [PATCH 0344/1345] pp3: Update PP3 driver version to NSS:16.07.0

commit  b0f9cc546a7b8e897a1f41058761e18c26f49d00 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Changes from prevoius version: NSS:16.06.1
- Fix bug in setting MAC address to interface before up first time.
- Fix bug in network interface down procedure. Wait for TXQs cleanup.
- Fix Xon/Xoff mechanism for Multi CoS environment.
- Disable TCP/UDP checksum offload on TX for nssX interfaces.
- Add support for checking PPNs keep alive status.
- Add configuration flag MV_PP3_COHERENCY_HARD_MODE_ONLY
to remove dma_map_single from the driver code.

Change-Id: I6aa6ebf547e9088168d206410014fc3cb7f1b595
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31192
Reviewed-by: Dovrat Zifroni <dovrat@marvell.com>
Tested-by: Dovrat Zifroni <dovrat@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../ethernet/marvell/pp3/platform/mv_pp3_version.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
index 02668a3..5d1c619 100644
--- a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
+++ b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
@@ -26,8 +26,8 @@
 static struct mv_pp3_version mv_pp3_driver_version = {
 	.name = "NSS",
 	.major_x = 16,
-	.minor_y = 6,
-	.local_z = 1,
+	.minor_y = 7,
+	.local_z = 0,
 	.debug_d = 0
 };
 
-- 
1.7.9.5

